package.json

```diff
@@ -80,6 +81,7 @@
     "lint:flow": "babel-node scripts/flow-check.js",
     "lint:other": "npm run prettier -- --check",
     "markdown": "md-magic --path \"starters/**/*.md\"",
+    "markdown-link-check": "find . -name \*.md -exec markdown-link-check {} \;",
```

Comment:
I think this is invalid JSON. It's failing CircleCI with:

`An unexpected error occurred: "/home/circleci/project/package.json: Unexpected token * in JSON at position 2667". `

Reply:
Fixed